src/app/zap-templates/templates/app/helper.js

```diff
@@ -414,6 +414,14 @@ function asChipZapType(type)
   return templateUtil.templatePromise(this.global, promise)
 }
 
+function asChipEnumType(type)
+{
+  if (type == 'Status') {
+    return 'chip::Protocols::InteractionModel::Status';
```

Comment:
I'm not sure whether this is the right place to do this mapping.  Seems like we'd want this in other places too, where currently we map ZAP types to Matter types.  @vivien-apple, where's the best place to do this?

Reply:
Where do we have cases where the type is 'status'? Didn't we remove those cases as per our discussion?

Reply:
We still have untouched status if you search  type="Status", which are existing one. In previous PR, I am not touching these existing one.

    <command source="server" code="0x40" name="EnhancedAddSceneResponse" optional="true" introducedIn="zll-1.0-11-0037-10" disableDefaultResponse="true">
      <description>
        Command description for EnhancedAddSceneResponse
      </description>
      <arg name="status" type="Status"/>
      <arg name="groupId" type="INT16U"/>
      <arg name="sceneId" type="INT8U"/>
    </command>